JenkinsFile

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/groovy
+
+node {
+    def appName = "bf-ia-broker"
+    def root = pwd()
+    def mvn = tool 'M3'
+    def golangTool = tool 'golang_1.7'
+    def zapHome = tool 'ZAProxy_v2_5_0'
+    def nodejs = tool 'NodeJS_6'
+    stage("Setup") {
+        deleteDir()
+        if(env.GITLAB_CREDS) {
+            git url: "${env.GIT_URL}", branch: "${env.GIT_BRANCH}", credentialsId: "${env.GITLAB_CREDS}"
+        } else {
+            git url: "${env.GIT_URL}", branch: "${env.GIT_BRANCH}"
+        }
+        withEnv([
+          "PATH+=${golangTool}/bin:${root}/gopath/bin",
+          "GOROOT=${golangTool}",
+          "GOPATH=${root}/gopath"
+        ]) {
+            sh """
+              mkdir -p ${root}/gopath/bin ${root}/gopath/pkg ${root}/gopath/src
+              go version
+            """
+        }
+        if(!params.SKIP_INTEGRATION_TESTS) {
+            if(!fileExists("integration-tests")) {
+                sh "mkdir -p integration-tests"
+            }
+            withCredentials([
+              file(credentialsId: "${POSTMAN_SECRET_FILE}", variable: "POSTMAN_FILE")
+            ]) {
+                withEnv([
+                  "PATH+=${nodejs}/bin",
+                  "NPM_CONFIG_CACHE=${root}/.npmcache",
+                  "HOME=${WORKSPACE}"
+                ]) {
+                    dir ("integration-tests") {
+                        if(env.GITLAB_CREDS) {
+                            git url: "${env.INTEGRATION_GIT_URL}", branch: "${env.INTEGRATION_GIT_BRANCH}", credentialsId: "${env.GITLAB_CREDS}"
+                        } else {
+                            git url: "${env.INTEGRATION_GIT_URL}", branch: "${env.INTEGRATION_GIT_BRANCH}"
+                        }
+                        sh """
+                          mkdir -p ${root}/.npmcache
+                          npm install newman@2
+                        """
+                    }
+                }
+            }
+        }
+    }
+    stage("Security Scan Pass/Fail") {
+        // Query threadfix and block if any highs/crits are found
+        /*sh "mkdir -p sspf-pkg"
+        dir("./sspf-pkg") {
+            withCredentials([[$class: 'StringBinding', credentialsId: "${env.THREADFIX_API_KEY}", variable: 'THREADFIX_KEY']]) {
+                sh "curl -L ${env.SSPF_PACKAGE} -o sspf.zip"
+                sh "unzip sspf.zip"
+                dir("./sspf-master/sspf") {
+                    sh "/bin/curl --fail --silent ${env.THREADFIX_URL}/rest/applications/${THREADFIX_ID}?apiKey=${THREADFIX_KEY} | ./sspf.py"
+                }
+            }
+        }*/
+    }
+
+    def appvers = sh(script: """git describe --long --tags --always | sed 's/\\./-/'g""", returnStdout: true)
+    appvers = appvers.trim()
+    def fullAppName = "${appName}-${appvers}"
+
+    stage("Archive") {
+        def archiveName = "${appName}.tgz"
```

Comment:
Did you check Nexus to make sure that it placed the artifact in the proper place?